guides/source/active_record_postgresql.md

```diff
@@ -290,23 +290,48 @@ irb> article.status_published?
 => false
 ```
 
-To add a new value before/after existing one you should use [ALTER TYPE](https://www.postgresql.org/docs/current/static/sql-altertype.html):
+To rename the enum you can use `rename_enum` along with updating any model
+usage:
 
 ```ruby
-# db/migrate/20150720144913_add_new_state_to_articles.rb
-# NOTE: ALTER TYPE ... ADD VALUE cannot be executed inside of a transaction block so here we are using disable_ddl_transaction!
-disable_ddl_transaction!
+# db/migrate/20150718144917_rename_article_status.rb
+def change
+  rename_enum :article_status, to: article_state
+end
+```
+
+```ruby
+# app/models/article.rb
+class Article < ApplicationRecord
+  enum state: {
```

Comment:
I believe we also need to update the `article_status` enum reference to `article_state` and rename `status` column to `state` 🤔